src/fusion_power_plant.h

```diff
@@ -59,27 +56,170 @@ class FusionPowerPlant : public cyclus::Facility  {
+    "range": [0, 1e299], \
+    "uilabel": "Buy quantity" \
+  }
+  double buy_quantity;
+
+  #pragma cyclus var { \
+    "doc": "Helium-3 output commodity Designation", \
+    "tooltip": "He-3 output commodity", \
+    "uilabel": "He-3 output commodity" \
+  }
+  std::string he3_outcommod;
+
+  #pragma cyclus var { \
+    "doc": "Recipe for blanket feed material", \
+    "tooltip": "Recipe for blanket feed material", \
+    "uilabel": "Recipe for blanket feed material" \
+  }
+  std::string blanket_inrecipe;
+
+  #pragma cyclus var { \
+    "doc": "Blanket waste commodity designation", \
+    "tooltip": "Blanket waste commodity", \
+    "uilabel": "Blanket waste commodity" \
+  }
+  std::string blanket_outcommod;
+
+  #pragma cyclus var { \
+    "default": 1000.0, \
+    "doc": "Initial mass of full blanket material", \
+    "tooltip": "Only blanket material mass, not structural mass", \
+    "units": "kg", \
+    "uitype": "range", \
+    "range": [0, 10000], \
+    "uilabel": "Initial Mass of Blanket" \
+  }
+  double blanket_size;
+
+  #pragma cyclus var { \
+    "default": 0.05, \
+    "doc": "Percent of blanket that gets recycled every blanket turnover period", \
+    "tooltip": "Defaults to 0.05 (5%), must be between 0 and 15%", \
+    "units": "dimensionless", \
+    "uitype": "range", \
+    "range": [0, 0.15], \
+    "uilabel": "Blanket Turnover Rate" \
+  }
+  double blanket_turnover_rate;
+
+  //Resource Buffers and Trackers:
   cyclus::toolkit::ResBuf<cyclus::Material> tritium_storage;
   cyclus::toolkit::ResBuf<cyclus::Material> tritium_excess;
   cyclus::toolkit::ResBuf<cyclus::Material> helium_excess;
   cyclus::toolkit::ResBuf<cyclus::Material> blanket_feed;
   cyclus::toolkit::ResBuf<cyclus::Material> blanket_waste;
 
-  //PyneIDs
-  const int tritium_id = 10030000;
+  cyclus::toolkit::MatlBuyPolicy fuel_startup_policy;
+  cyclus::toolkit::MatlBuyPolicy fuel_refill_policy;
+  cyclus::toolkit::MatlBuyPolicy blanket_fill_policy;
 
-  //Compositions:
-  const cyclus::CompMap T = {{tritium_id, 1}};
-  const cyclus::Composition::Ptr tritium_comp = cyclus::Composition::CreateFromAtom(T);
+  cyclus::toolkit::MatlSellPolicy tritium_sell_policy;
+  cyclus::toolkit::MatlSellPolicy helium_sell_policy;
+  cyclus::toolkit::MatlSellPolicy blanket_waste_sell_policy;
 
-  //Materials:
-  cyclus::Material::Ptr sequestered_tritium = cyclus::Material::CreateUntracked(0.0, tritium_comp);
+  cyclus::toolkit::TotalInvTracker fuel_tracker;
+  cyclus::toolkit::TotalInvTracker blanket_tracker;
+
+  //Functions:
+  void CycleBlanket();
+  bool CheckOpeartingConditions();
+  bool CheckInitialStartupCondition();
+  void SequesterTritium();
+  void OperateReactor();
+  void CycleBlanket();
+  void DecayInventories();
+  void ExtractHelium();
+  void MoveExcessTritiumToSellBuffer();
+
+  private:
+    bool initial_startup_condition_met = false;
+    double fuel_usage_atoms;
+    double fuel_usage_mass;
+    const double seconds_per_year = 2629846*12;
```

Comment:
I feel as though we need to make this a constant available in Cyclus for uniformity...  and it should possibly be based on a days
```
seconds_per_year = 365.25 * 24 * 3600
seconds_per_month = seconds_per_year / 12
```

Reply:
This is a whole rabbit hole. Cyclus seems to use some variety of rounded Gaussian year which is ~365.256 days long, and I've never fully understood why that is, or what the rounding convention was (it's not exactly that which is just so confusing to me). This is defined in context.h as `const uint64_t kDefaultTimeStepDur = 2629846` with no comments saying why that is. Happy to discuss further.

Reply:
Yeah - there is probably a good reason to clarify/fix this for all of Cyclus - we should launch a convo in an issue and/or slack to get Katie & Amanda's input (among others)

Reply:
I can do both now!